epub34/authoring/index.html

```diff
@@ -6347,8 +6292,8 @@ <h4>Script inclusion</h4>
 								if a future update adds the concept.</p>
 						</div>
 
-						<p>[=EPUB creators=] should note that [=reading systems=] are required to behave as though a
-							unique <a data-cite="html#origin">origin</a> [[html]] has been assigned to each [=EPUB
+						<p>Note that [=reading systems=] are required to behave as though a unique <a
+								data-cite="html#origin">origin</a> [[html]] has been assigned to each [=EPUB
```

Comment:
```suggestion
						<p>Note that [=reading systems=] are required to behave as though a unique <a
								data-cite="html#origin">origin</a> [[html]] had been assigned to each [=EPUB
```

Isn't that the correct grammatical form? But I may be wrong.

Reply:
The behaviour continues into the present, so "has been" sounds more accurate here. "had been" makes it sound like the assignment of an origin happened but now there's a new post-assignment state.

Reply:
I yield to your English knowledge :-)